### DISCUSSION TITLE
Restyle floatingRefreshSection

### DISCUSSION THREAD

**Discussion by pkrasicki** [FreeTubeApp/FreeTube#6293] [General]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

The floating refresh section in FreeTube 0.23 (modified in #4380) obscures some of the content while scrolling.

![image](https://github.com/user-attachments/assets/be5eeb41-5e51-4fb6-b432-fc6bf4f12f12)

### Proposed Solution

Remove most of the background and reposition the text, so that it obscures less content.

![refresh-section-light](https://github.com/user-attachments/assets/235627ef-c77b-4212-96cd-751d93c2c50c)

[refresh-section.webm](https://github.com/user-attachments/assets/1fcf3683-8866-4905-9b5c-1dc69b507b61)

### Alternatives Considered

None.

### Issue Labels

visual improvement

### Additional Information
CSS code (needs testing on mobile):
```
.floatingRefreshSection
{
	background-color: transparent;
	box-shadow: none;
	border-inline-start: none;

	.lastRefreshTimestamp
	{
		background-color: var(--card-bg-color);
		position: relative;
		top: -12px;
	}
}
```

**Comment by pkrasicki** [comment]
I forgot that unlike my design FreeTube has different background color for the card and for the page background, so this change won't look as good as on my screenshots. But maybe it's still worth trying?

Edit:
Here is what it would actually look like in FreeTube. I think it looks good:
![refresh-section-ft](https://github.com/user-attachments/assets/f688766c-43d2-47ad-b0e7-067d0e36f4de)

![refresh-section-ft2](https://github.com/user-attachments/assets/61ea97bf-1bde-47c3-b528-f1c76f80a9d2)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Converted this to a general discussion as this is more a subjective topic than a feature like with allot of UI topics

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I tested allot of screen sizes and i dont see the refresh button obscuring content. Could you maybe provide a screencap of it?

**Comment by pkrasicki** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc It's not the about the button, it's the whole bar. Your video shows this perfectly (just ignore the bug that it was showing): https://github.com/FreeTubeApp/FreeTube/pull/4380#issuecomment-2063764486.

The bar is not transparent, so it obscures the thumbnails and other content while scrolling. To fix that, I made it transparent and moved the text a bit.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Ah ok, so i thought this over a little bit but i fail to see why this restyling is needed. It doesnt obscure content imo + the bar has a clear purpose.

The top bar here for example does exactly the same thing. It isnt jarring and in the way or something like that. Would like to hear from others but to me this feels like a non issue

https://github.com/user-attachments/assets/621aa624-fe20-4123-821c-bd3cae14b7b9

**Reply by pkrasicki** [reply]
>It doesnt obscure content imo + the bar has a clear purpose.

What purpose does it have? We are displaying a short string of text and a button. Do those things need to take up so much space? Actually, do they even need to float at all? Pressing the refresh button will cause your scroll position to reset, so what's the point of it following you as you scroll? The floating text seems more useful to me, but subscriptions page doesn't refresh automatically, so do I really need to see this information at all times? Do other video apps do this?

>The top bar here for example does exactly the same thing. It isnt jarring and in the way or something like that. Would like to hear from others but to me this feels like a non issue

Any floating bar takes up precious space from what you actually want to see - the content. So I think it's best to avoid it if possible. Does it make sense on GitHub? I don't know, it certainly provides context telling you what conversation you're looking at, so that you don't forget. That might be useful, but GitHub is probably not a good UI reference in general. Are there any video apps that do this? I'm not sure, because I don't have an account on any of them to look at a list of subscriptions. So if anyone could check and provide screenshots for reference, that would be very helpful.

But don't forget that FreeTube already has one such floating bar - the nav bar with search and buttons (which is useful). This section is another bar in addition to that.

**Reply by pkrasicki** [reply]
Alternative proposal without floating:

![refresh-section3](https://github.com/user-attachments/assets/771fedb3-3fa9-4039-a259-a7cc8bea2772)

The button's background color can be brighter if you want. I think it should be subtle, but maybe I made it too subtle.
